Index: src/com/ellane/model/Characters.java
===================================================================
diff --git a/src/com/ellane/model/Characters.java b/src/com/ellane/model/Characters.java
deleted file mode 100644
--- a/src/com/ellane/model/Characters.java	
+++ /dev/null	
@@ -1,17 +0,0 @@
-package com.ellane.model;
-
-public enum Characters {
-    MALE_SOLDIER("soldier"),
-    KID("kid"),
-    DOG("dog");
-
-    private final String characterType;
-
-    Characters(String characterType) {
-        this.characterType = characterType;
-    }
-
-    public String getCharacterType() {
-        return characterType;
-    }
-}
\ No newline at end of file
